ci/docker/install/ubuntu_tutorials.sh

```diff
@@ -24,5 +24,9 @@ set -ex
 apt-get update || true
 apt-get install graphviz
 
-# sckit-learn past version 0.20 does not support python version 2 and 3.4
-pip3 install jupyter matplotlib Pillow opencv-python scikit-learn graphviz==0.8.4 tqdm mxboard scipy gluoncv
+pip3 install graphviz==0.8.4 tqdm mxboard
+pip3 install jupyter
+pip3 install matplotlib Pillow opencv-python
+pip3 install scipy gluoncv
+pip3 install scikit-learn
+
```

Comment:
unrelated to graduation name change but looks good to me regardless.

Reply:
Building of the docker container started hanging while installing all the PIP packages because of dependency chains, but splitting them up works fine.